app/sign-out/controllers.js

```diff
@@ -0,0 +1,21 @@
+const { IS_PRODUCTION, SERVER_HOST, AUTH } = require('../../config')
+
+function buildRedirectUrl () {
+  const protocol = IS_PRODUCTION ? 'https' : 'http'
+  return `${protocol}://${SERVER_HOST}`
```

Comment:
I think we may need a more generic way to figure out the protocol too. I’ve used some middleware in other apps to do this so it can be shared by things like building canonical URLs or asset paths.

Reply:
I'm surprised Express doesn't provide this already, but I think it's outside the scope of this feature.

Reply:
It does have this: https://expressjs.com/en/api.html#req.protocol

But there are some issues with behind a proxy.